.github/scripts/build-default-workloads.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-from fabric.api import *
+from fabric.api import prefix, settings, run, execute # type: ignore
```

Comment:
So this ignores all the typing information coming out of functions in fabric.api right? I've looked into this in the past,  and don't remember seeing any way to get the types out of fabric.api, but maybe it's worth looking into again?

Reply:
Correct. Fabric doesn't have any types. I also since we use a hacked version of Fabric (fab-classic) I don't expect it to add types anytime soon. We use this "hack" across the codebase in the interim.